doc/source/installation.rst

```diff
@@ -1,140 +1,209 @@
-    Microsoft Visual C++ 14.0 installed. If your installation
-    fails with the error message:
+    - If you encounter errors on **Windows** related to **Microsoft Visual C++ 14.0**, install the required **Build Tools for Visual Studio** from: `Microsoft Visual Studio Downloads`_. 
+    - If you encounter any other issues following these instructions, seek help on `GitHub Discussions (Installation)`_.    
 
-        error: Microsoft Visual C++ 14.0 is required. Get it with "Build Tools for Visual Studio": https://visualstudio.microsoft.com/downloads/
+.. warning::
+    :ref:`mamba-installation` and :ref:`pip-installation` installations **do not include** external programs such as `HOLE`_.
 
-    Try installing Build Tools for Visual Studio from
-    https://visualstudio.microsoft.com/downloads/ (scroll
-    down to the Tools for Visual Studio section).
+.. _mamba-installation:
 
+mamba/conda 
+===========
 
-If you encounter any issues following these instructions, please
-ask for help on `GitHub Discussions (Installation)`_.
+For most users, `mamba`_ is the **recommended** way to install MDAnalysis. It is a faster drop-in 
+replacement for `conda`_ and efficiently handles dependencies. We also recommend creating a new 
+environment for MDAnalysis to ensure a clean and isolated installation.
 
-.. _`GitHub Discussions (Installation)`: https://github.com/MDAnalysis/mdanalysis/discussions/categories/installation
+MDAnalysis supports two common ways to install with `mamba`_:
 
-conda
-=====
+**1. Recommended: Use Mambaforge**
 
-If you use ``conda`` to manage your Python environment, we highly recommend creating a new environment for MDAnalysis.
-This will ensure that you have a clean installation of MDAnalysis and its dependencies, and will not interfere with other packages you may have installed.
-We further recommend that you install and use ``mamba``, a faster drop-in replacement for ``conda``.
+`Mambaforge`_ is a minimal `conda`_ distribution that includes `mamba`_ by default, uses the `conda-forge`_ channel 
+(preferred for MDAnalysis) and avoids clutter from `Anaconda`_ base packages.
+
+To install MDAnalysis with `Mambaforge`_, first download the `mambaforge installer`_ and then run: 
 
 .. code-block:: bash
 
-    conda create --name mdanalysis
-    conda activate mdanalysis
-    conda install -c conda-forge mamba
+    mamba create -n mdanalysis mdanalysis
+    mamba activate mdanalysis
 
-To install the latest stable version of MDAnalysis via ``conda``, use the following command. This installs all dependencies needed for full analysis functionality (excluding external programs such as `HOLE`_):
+To install the :ref:`test suite <mdanalysistests>` (optional, ~90 MB):
 
 .. code-block:: bash
 
-    mamba install -c conda-forge mdanalysis
+    mamba install -c conda-forge MDAnalysisTests
+
+For more installation options (including Windows installers), see the `official mamba installation guide`_.
```

Comment:
This sentence seems out of context. If it's about mambaforge, put it there (perhaps in a footnote?)